src/rtc/h5_prep.py

```diff
@@ -301,15 +301,29 @@ def save_orbit(orbit, orbit_group, orbit_file_path):
         'referenceEpoch',
         data=np.string_(orbit.reference_epoch.isoformat()))
 
-    # Orbit source/type    
-    if 'RESORB' in orbit_file_path:
-        orbit_type = 'RES restituted orbit'
-    elif 'POEORB' in orbit_file_path:
-        orbit_type = 'POE precise orbit'
-    else:
-        orbit_type = 'Undefined'
-
-    d = orbit_group.require_dataset("orbitType", (), "S25",
+    # Orbit source/type
+    if isinstance(orbit_file_path, str):
+        orbit_file_basename = os.path.basename(orbit_file_path)
+        if 'RESORB' in orbit_file_basename:
+            orbit_type = 'RES restituted orbit'
+        elif 'POEORB' in orbit_file_basename:
+            orbit_type = 'POE precise orbit'
+        else:
+            orbit_type = 'Undefined'
+
+    elif isinstance(orbit_file_path, list):
+        orbit_type_list = []
+        for individual_orbit_file in orbit_file_path:
+            if 'RESORB' in individual_orbit_file:
+                orbit_type_individual = 'RES restituted orbit'
+            elif 'POEORB' in individual_orbit_file:
+                orbit_type_individual = 'POE precise orbit'
+            else:
+                orbit_type_individual = 'Undefined'
+            orbit_type_list.append(orbit_type_individual)
+        orbit_type = '; '.join(orbit_type_list)
```

Comment:
```suggestion
    # Orbit source/type
    orbit_type = 'Undefined'
    if isinstance(orbit_file_path, str):
        orbit_file_basename = os.path.basename(orbit_file_path)
        if 'RESORB' in orbit_file_basename:
            orbit_type = 'RES restituted orbit'
        elif 'POEORB' in orbit_file_basename:
            orbit_type = 'POE precise orbit'

    elif isinstance(orbit_file_path, list):
        orbit_type_list = []
        for individual_orbit_file in orbit_file_path:
            if 'RESORB' in individual_orbit_file:
                orbit_type_individual = 'RES restituted orbit'
            elif 'POEORB' in individual_orbit_file:
                orbit_type_individual = 'POE precise orbit'
            else:
                orbit_type_individual = 'Undefined'
            orbit_type_list.append(orbit_type_individual)
        orbit_type = '; '.join(orbit_type_list)
```
@seongsujeong , this change looks good to me. Only minor comment is that if you have this `if/elif` statement, you also need to consider the case in which `orbit_file_path` is not an `str` or a `list`. I suggest you move `             orbit_type = 'Undefined'` to the beginning so you make sure `orbit_type` is initialized.